src/views/PopupView.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from "react";
 import * as cameraControlsImg from "../../assets/images/camera-controls.png";
 import * as selectionControlsImg from "../../assets/images/selection-controls.png";
-import * as timelineControlsImg from "../../assets/images/timeline-controls.png";
```

Comment:
With Javi's PR being recently merged, it looks like this was accidentally deleted. This should be added back in